handleAddition.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require_once 'functions.php';
+databaseSetUp();
+//validates Date
+$datePattern  = '/[1-9][0-9][0-9]{2}-([0][1-9]|[1][0-2])-([1-2][0-9]|[0][1-9]|[3][0-1])/';
```

Comment:
I applaud your use of a regex, but it would be better to use PHP's built in `checkdate` for this: https://www.php.net/manual/en/function.checkdate.php